packages/angular/README.md

```diff
@@ -0,0 +1,22 @@
+# stencil-ds-angular-template
```

Comment:
I'm assuming this readme will be updated? Is there a ticket for it?

Reply:
updated. It might miss few things but I figure I'll update the docs when the bug described in the description is fixed